client/app/components/routes/access-requests/form/UserOperatorReview.tsx

```diff
@@ -50,7 +55,11 @@ export default function UserOperatorReview({
         "PUT",
         "",
         {
-          body: JSON.stringify({ status, user_operator_id: userOperatorId }),
+          body: JSON.stringify({
+            role: UserOperatorRoles.ADMIN,
```

Comment:
@andrea-williams here we set the role from the default `pending` to `admin` when they approve the prime admin request.